src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(double length, double width, String color) {
+        this.length = length;
+        this.width = width;
+        this.setColor(color);

```

Comment:
```suggestion
        super(color);
```
create a constructor in figure